configs/projects/puppet-agent.rb

```diff
@@ -92,6 +92,7 @@
   # Cross-compiled Linux platforms
   platform_triple = "powerpc-linux-gnu" if platform.is_huaweios?
   platform_triple = "s390x-linux-gnu" if platform.architecture == "s390x"
+  platform_triple = "arm-linux-gnueabihf" if platform.name == 'debian-8-armhf'
```

Comment:
This should also be added to the Vanagon wishlist; it's not really project specific so we should move it into Vanagon. Obviously out of scope for this PR.